tests/istio.mk

```diff
@@ -46,28 +46,31 @@ E2E_ARGS += $(if ifeq($V,1),-alsologtostderr -test.v -v 2)
 E2E_ARGS += ${MINIKUBE_FLAGS}
 E2E_ARGS += --istioctl ${GOPATH}/bin/istioctl
 
-
+# A make target to generate all the YAML files
+generate_yaml:
+	./install/updateVersion.sh >/dev/null 2>&1
```

Comment:
may not hurt to not send to dev/null in case there is some error output there (yes it is floody but it's better to know something went wrong)

also the dependency should be : install/kubernetes/istio.yaml istio.VERSION
so it doesn't keep get regenerated

Reply:
doesn't matter.. its use and throw.. and its on circleci.. And the amount of output is huge. I prefer to see actual test output more than sed outputs.. We can optimize this later and redirect to file and examine if needed..